components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIConsumer.java

```diff
@@ -146,7 +146,15 @@
      * @throws APIManagementException
      */
     public SubscribedAPI getSubscriptionById(int subscriptionId) throws APIManagementException;
-    
+
+    /** returns the SubscribedAPI object which is related to the UUID
+     *
+     * @param uuid UUID of Subscription
+     * @return
+     * @throws APIManagementException
+     */
+    public SubscribedAPI getSubscriptionByUUID(String uuid) throws APIManagementException;
```

Comment:
JavaDocs for the return statement on this method is missing. A public API should always have a proper Java Docs. Fix these issue across all the PR's.